### PR TITLE
Instruction on how to install poshgit via Chocolatey.

### PR DESCRIPTION
**Pull request by zaylon10**

### CODE REVIEW

**Comment by thatoldplatitude** [comment]
nit: have branch up to date with main

**Review by eld120 — APPROVED** [review]

**Review by stephenlauck — CHANGES_REQUESTED** [review]
What if we convert this README to a powershell ps1 that we could just run?

**Review by stephenlauck — APPROVED** [review]
ok to merge just for historical purpose.

File: windows-poshgit.ps1 [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  echo "Installing chocolatey"
+  Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.servicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))
+}
+
+
+# git should be installed
+
+if (get-command git)
+{
+  echo "chocolatey already installed"
```
Comment (eld120) [inline windows-poshgit.ps1:20]: @zaylon10  let's change the terminal output on line 20 from "chocolatey already installed" to "git already installed"

**Comment by ChrisForti** [comment]
This not ready for merge, needs to be tested on another windows machine. We were getting intermittent errors, saying that poshgit is not cmdlet function.

**Review by stephenlauck — APPROVED** [review]

**Pull request merged**